boards/nxp/frdm_rw612/doc/index.rst

```diff
@@ -188,8 +188,22 @@ frdm_rw612 platform supports the monolithic feature. The required binary blob
 ``<zephyr workspace>/modules/hal/nxp/zephyr/blobs/rw61x_sb_wifi_a2.bin`` will be linked
 with the application image directly, forming one single monolithic image.
 
+RTC Sub-Second Counter
+======================
+
+To use the RTC sub-second counter which is clocked at a 32kHZ rate, you first need to make the
+following modifications to the board hardware:
+
+1. Move the short on SJ21 from 1 and 2 to short 2 and 3.
+2. Move the short on SJ22 from 1 and 2 to short 2 and 3.
```

Comment:
```suggestion
1. Move the jumper on SJ21 from pins 1-2 to 2-3.
2. Move the jumper on SJ22 from pins 1-2 to 2-3.
```
my preference

Reply:
These are not jumpers on the board. They are physical shorts. Should I leave it unchanged?

Reply:
Thanks @mmahadevan108.  "SJ" is a Solder Jumper.  But we could change to this to clarify
```suggestion
1. Move the solder jumper on SJ21 from pads 1-2 to 2-3.
2. Move the solder jumper on SJ22 from pads 1-2 to 2-3.
```